### ISSUE TITLE
Lines printed on import

### ISSUE THREAD

**Issue by nickolivotrainer** [davebelais/gtin#1]
@davebelais - When importing the library, 150 lines are printed 
(1, 11000, 8800.0, -351200.0)
...
(150, 1650000, 1320000.0, 960000.0)

is there a way to suppress that?

(This module is very helpful by the way, thanks for making it! :) )

**Comment by davebelais** [comment]
Hi @nickolivotrainer - 

That's odd, can you check and tell me what version you are using? I don't see anything that would do that in the current version...

```python
import pkg_resources
print(pkg_resources.get_distribution('gtin').version)
```
-David

**Comment by nickolivotrainer** [comment]
Sure, it's 0.1.12

**Comment by davebelais** [comment]
Hi @nickolivotrainer -

I can't seem to recreate you experience, and am at a loss at to what would cause that, as there are no `print` commands [in the module](https://github.com/davebelais/gtin/search?q=print&type=Code) (only in the tests and distribution script)... If you want to attach source files, I will take a look at them, however.

-David

**Comment by nickolivotrainer** [comment]
Thanks, David. I didn't see any print commands in there either, which was throwing me. I've attached the files, please let me know if there's any other info I can provide. 
[gtin.zip](https://github.com/davebelais/gtin/files/3933188/gtin.zip)

**Comment by davebelais** [comment]
Hi Nick-

Pardon, I meant source files for the script you were importing `gtin` into... A gist would work as well, if you can recreate your issue in the context of a single-file script. Also, if you could output a requirements.txt file (`pip3 freeze -r > requirements.txt` from the command line) for the environment you are using, and let me know what python version—that would help as well. 

-David

**Comment by nickolivotrainer** [comment]
Hi David
I'm running python 2.7.13 on Windows 10. As for the script, if I launch IDLE and run import gtin, the behavior occurs. Here's a 10 second vid showing that - https://www.screencast.com/t/iO2d8v7kaH - if I import other modules, the behavior doesn't occur.

**Comment by davebelais** [comment]
Hi Nick-

Could you run `pip2.7 freeze > requirements.txt` and send me the file generated? That will tell me all of the libraries you've got installed and their version numbers.

Thanks!

**Comment by nickolivotrainer** [comment]
Hi David

Sure thing, here you go. 
[requirements.txt](https://github.com/davebelais/gtin/files/3939763/requirements.txt)

**Comment by davebelais** [comment]
Hi Nick-

...I wasn't able to replicate the issue, so I'm not sure this will do anything--but try updating the module (I uploaded a new minor version), and let me know if you still get the same issue: `pip2.7 install --upgrade gtin`.

-David

**Comment by nickolivotrainer** [comment]
Hi David

After some digging, I figured out what was causing this. Turns out it was nothing to do with the gtin library - gtin's __init__.py calls install_aliases(), and within install_aliases() there's this try statement
    # Patch the test module so it appears to have the same structure on Py2 as on Py3
    try:
        import test
    except ImportError:
        pass
I'm guessing that most systems don't have the test module, but I had written one of my own and that module does write out 150 lines of data (I wrote it so long ago, I'd forgotten all about it). So install_aliases() was calling my test library, which wrote the data. 

So, this isn't anything you need to worry about, and I'm sorry for wasting your time. Thanks again for looking into this and helping me out.

Best
Nick

**Comment by davebelais** [comment]
Resolved w/ version 1.0, which drops python 2x support
